src/components/Pill/__tests__/Pill.spec.tsx

```diff
@@ -24,47 +25,61 @@ describe('<Pill> component', () => {
                 {childrenMock}
             </Pill>
         );
-
-        clickPillButton = () => wrapper.find('.PillButton__pill').simulate('click');
     });
 
     afterEach(() => {
         jest.resetAllMocks();
     });
 
     it('should render correctly', () => {
-        expect(toJson(wrapper)).toMatchSnapshot();
-        expect(wrapper.find('PillButton')).toHaveLength(1);
-        expect(wrapper.find('PillDropdown')).toHaveLength(0);
+        expect(view.container).toMatchSnapshot();
+        expect(screen.getByRole('button', { name: 'Pill content' })).toBeInTheDocument();
+        expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
     });
-    it('should open dropdown when button is clicked', () => {
-        clickPillButton();
-        expect(toJson(wrapper)).toMatchSnapshot();
-        expect(wrapper.find('PillButton')).toHaveLength(1);
-        expect(wrapper.find('PillDropdown')).toHaveLength(1);
+
+    it('should open dropdown when button is clicked', async () => {
+        const user = userEvent.setup();
+        await user.click(screen.getByRole('button', { name: 'Pill content' }));
```

Comment:
I think it is better the way you did in some other places, where you used the variable and not repeated the params. `{ name: 'Pill content' }` => `{ name: nameMock }`

Reply:
:+1: